### DISCUSSION TITLE
Aluminium crystal plasticity parameters

### DISCUSSION THREAD

**Discussion by bibiboom89** [neperfepx/FEPX#110] [Q&A] [answered]
Bonjour Romain Quey,

I'm trying to perform macro-scale simulations using FEPX of aluminum alloy 6000-series (Al6082 to be more precise).
While looking into the applications of FEPX I saw a couple of articles concerning aluminum of which you are the author.
But I don't think your parameters fit for what I'm looking for, but I was wondering if maybe you could guide me toward the right article.

I'm just trying to simulate the early stage of plasticity. 
It doesn't have to be precise parameters. 
Just parameters suited for tensile and/or cyclic tests.
The goal is to locate where the stress concentrations are within the mesh made from an EBSD map. 

Thank you for everything,
Regards,
Bibi

**Comment by bibiboom89** [comment]
Hi again :)

After some research, I tried two different sets of parameters from two different papers, but both are giving weird results.
When I plot the strain-stress curve, plasticity starts to occur very early, which is not consistent with what it is presented in the papers where the parameters come from:
![Screenshot from 2024-03-18 14-13-39](https://github.com/neperfepx/FEPX/assets/63070505/cbaf8497-a765-49e6-bdaf-dd1637a43b91)

I'm wondering then that I might be doing something wrong in the configuration file? 
You can find in the attachment the file [simulation.config.txt](https://github.com/neperfepx/FEPX/files/14641142/simulation.config.txt) I used for the second set of parameters, which can be found in: https://doi.org/10.1016/j.ijplas.2008.11.003.
I'm still on the version 1.3.0 of the software (I'm not able to make the version 2 to work on my server).
Could you help me please to find what I've done wrong?


Bonus questions :
1. How can I extract a slip system's direction and normal ? 
I couldn't find it in the documentation.
I could calculate it myself, but I don't know either to which slip system corresponds the slip system 'i' from the result file.
2. In parallel, I'm also working on Titanium. You sheared me an excellent article for the material parameters (thank you again): https://doi.org/10.1016/j.actamat.2017.06.059
I'm confused on how to write the parameters from that article in the FEPX configuration file.
I didn't find how to define different slip systems parameters (basal (b), prismatic (p), pyramidal (pi)).
Could you guide with that please?

Thank you for everything,
Regards,
Bibi

**Reply by mkasemer** [reply, marked as answer]
The stress strain behavior above broadly agrees with what is presented in Figure 6 of the first paper, so I don't see the issue with the behavior observed with the parameters used.

The slip normals and directions are presented in the FEPX manual.

Definition of basal, prismatic, and pyramidal slip strengths are also described in the manual when specifying `g0` for HCP materials.
